tests/framework/e2e/cluster.go

```diff
@@ -142,13 +149,11 @@ type EtcdProcessClusterConfig struct {
 
 	SnapshotCount int // default is 10000
 
-	ClientTLS             ClientConnType
-	ClientCertAuthEnabled bool
-	IsPeerTLS             bool
-	IsPeerAutoTLS         bool
-	IsClientAutoTLS       bool
-	IsClientCRL           bool
-	NoCN                  bool
+	Client        ClientConfig
+	IsPeerTLS     bool
+	IsPeerAutoTLS bool
+	IsClientCRL   bool
```

Comment:
Why do you still keep this field? Isn't it included in ClientConfig? 
```suggestion
	IsClientCRL   bool
```